zio-http/src/main/scala/zhttp/service/server/content/handlers/ServerResponseHandler.scala

```diff
@@ -65,10 +69,15 @@ private[zhttp] trait ServerResponseHandler[R] {
   /**
    * Releases the FullHttpRequest safely.
    */
-  private def releaseRequest(jReq: FullHttpRequest): Unit = {
-    if (jReq.refCnt() > 0) {
-      jReq.release(jReq.refCnt()): Unit
+  private def releaseRequest(jReq: HttpRequest)(implicit ctx: Ctx): Unit = {
+    jReq match {
+      case req: FullHttpRequest =>
+        if (req.refCnt() > 0) {
+          req.release(req.refCnt()): Unit
+        }
+      case _                    => if (!ctx.channel().config().isAutoRead) ctx.read(): Unit
```

Comment:
Why do we need this change?

Reply:
Because when you have objectAggregator autoread will be enabled. we need to do this if auto read is `false` to read the next request.

Reply:
We shouldn't do this here. Release request should only release the request. AutoRead should be reset in the ResponseHandler.

Reply:
We can revert this method completely here.